File: C/Assignment09/Program5.c
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// File name :   Program5.c
// Description : Program to which accept number from user & count frequency in such a digits which are less than 6.
// Author :      Pallavi Omprakash Malewar
// Date :        20/06/2025
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

/*
Input : 2395
Output : 3

Input : 1018
Output : 3

Input : 9440
Output : 3

Input : 96672
Output : 1
*/

#include<stdio.h>

int Count(int iNo)
{
    int iDigit = 0 , iCnt = 0 ;

    while(iNo != 0)
    {
        iDigit = iNo % 10;
        if(iDigit < 6)
        {
            iCnt++;
        }
        iNo = iNo / 10;
    }
    return iCnt;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    iRet = Count(iValue);

    printf("%d",iRet);

    return 0;
}

File: JAVA/Assignment 1/Program5.java
/////////////////////////////////////////////////////////////////////////////////////////
//                                                                                     //
// Problem Statement : Write a program which accept number from user and count         //
//                     frequency of such a digits which are less than 6.               //
// Author : Pallavi Omprakash Malewar                                                  //
// Date : 30 Jun 2026                                                                  //
//                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////



/*

    Input: 2395
    Output : 3

    Input: 1018
    Output : 3

    Input: 9440
    Output : 3

    Input: 96672
    Output : 1

*/


import java.util.Scanner;

class Program5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.print("Enter the Number : ");
        int No = sobj.nextInt();
        
        int freq = CountTwo(No);

        System.out.println(freq);
        
    }

    public static int CountTwo(int No)
    {
        int Digit = 0, Count = 0;

        if(No < 0)
        {
            No = -No;
        }

        if(No == 0)
        {
            return 1;
        }

        while(No != 0)
        {
            Digit = No % 10;
            if(Digit < 6)
            {
                Count++;
            }
            No /= 10;
        }
        return Count;
    }
}